### PR TITLE
Helm release

### LINKED ISSUE #4: Publish Helm charts on GitHub pages

### PR DESCRIPTION
**Pull request by MichaelKora**
Fixes #4

### CODE REVIEW

File: charts/README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ## Install
 
 ```
-helm repo add bakdata-kafka-connect-resetter https://raw.githubusercontent.com/bakdata/kafka-connect-resetter/<branch_name>/charts/
-helm install bakdata-kafka-connect-resetter/kafka-connect-resetter
+helm repo add bakdata-kafka-connect-resetter https://bakdata.github.io/kafka-connect-resetter/
+helm install bakdata-kafka-connect-resetter/kafka-connect-resetter 
 ```
 
 ## Development
```
Comment (philipp94831) [inline charts/README.md:10]: Please remove this notice

**Review by philipp94831 — CHANGES_REQUESTED** [review]
Please remove the tgz file https://github.com/bakdata/kafka-connect-resetter/blob/main/charts/kafka-connect-resetter/kafka-connect-resetter-0.1.0.tgz

File: .github/workflows/heal-release.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: Release Helm Charts
+on:
+  workflow_dispatch:
+
+jobs:
+  call-workflow-passing-data:
+    name: Release & Publish Helm chart
+    uses: bakdata/ci-templates/.github/workflows/helm-multi-release.yaml@main
```
Comment (yannick-roeder) [inline .github/workflows/heal-release.yaml:8]: Please use a tag version instead of main

**Comment by yannick-roeder** [comment]
What about the lint config file?

File: .github/workflows/heal-release.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: Release Helm Charts
+on:
+  workflow_dispatch:
```
Comment (yannick-roeder) [inline .github/workflows/heal-release.yaml:3]: We want to automatically run on tag branches, not manually

File: .github/workflows/heal-release.yaml (hunk 2 above)
Comment (MichaelKora) [reply .github/workflows/heal-release.yaml:8]: https://github.com/bakdata/ci-templates should release a new tag here?

File: charts/README.md (hunk 1 above)
Comment (MichaelKora) [reply charts/README.md:10]: what do you mean here

Comment (yannick-roeder) [reply charts/README.md:10]: The `Development` notice for packing it below is not needed anymore since your pipeline is doing it

**Review by yannick-roeder — APPROVED** [review]

**Pull request merged**